File: android/app/src/main/java/com/example/exercicio/model/UserBO.kt
package com.example.exercicio.model

class UserBO {

    var id: Int? = 0
    var email: String? = null
    var password: String? = null

    constructor(id: Int, email: String, password: String) {
        this.id = id;
        this.email = email;
        this.password = password;
    }

    constructor(email: String, password: String) {
        this.email = email;
        this.password = password;
    }

}

File: ios/exercicios/exercicios/Model/UserBO.swift
//
//  UserBO.swift
//  exercicios
//
//  Created by mcastro on 24/04/2019.
//  Copyright © 2019 mcastro. All rights reserved.
//

import Foundation

class UserBO {
    var id: Int?
    var email: String
    var password: String
    
    init(id: Int, email: String, password: String){
        self.id = id
        self.email = email
        self.password = password
    }
    
    init(email: String, password: String){
        self.email = email
        self.password = password
    }
}
